libs/service-portal/assets/src/screens/Lookup/Lookup.tsx

```diff
@@ -320,6 +324,26 @@ const Lookup = () => {
                   title: formatMessage(messages.weightedWLTPCo2),
                   value: String(weightedco2Wltp),
                 },
+                engine && {
+                  title: formatMessage(messages.engineType),
+                  value: engine,
+                },
+                hasOperatorNames ||
+                allOperatorsAreAnonymous ||
+                (someOperatorsAreAnonymous && hasOperatorNames)
+                  ? {
+                      title: formatMessage(messages.operator),
+                      value: (
+                        <LookupOperator
+                          names={operatorNames?.length ? operatorNames : []}
+                          allOperatorsAreAnonymous={!!allOperatorsAreAnonymous}
+                          someOperatorsAreAnonymous={
+                            !!someOperatorsAreAnonymous
+                          }
+                        />
+                      ),
+                    }
+                  : undefined,
```

Comment:
yeah but then it would be wrong, someOperatorsAreAnonymous is only relevant when the array hasOperatorNames has items, if someOperatorsAreAnonymous is the only boolean returning a true value, then something is wrong.